Gemfile.lock

```diff
@@ -369,4 +373,4 @@ RUBY VERSION
    ruby 2.4.1p111
 
 BUNDLED WITH
-   1.16.2
+   2.0.2
```

Comment:
Did we mean to do this? @alexander-mathieu @bplantico

Reply:
Seems fine, if anything goes wrong it'll probably be with Heroku!